lifechanger/src/app/projects/project-card/project-card.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="project-card__container">
+  <img
+    [src]="projectCardsInfo.imageAndAlt.image"
+    [alt]="projectCardsInfo.imageAndAlt.alt"
+  />
+  <!-- NEEDS alt, THINK OF THAT IMPLEMENTATION -->
```

Comment:
is this cooment is required, if no - please remove it

Reply:
Thank you, dear Mentor, for the remark, I repaired:)
I left that comment to not forget about the alt attribute, it is not necessary now👍